modules/preview-react/segmented-control/stories/SegmentedControl.stories.mdx

```diff
@@ -0,0 +1,205 @@
+import {Meta} from '@storybook/addon-docs';
+
+import {SegmentedControl} from '@workday/canvas-kit-preview-react/segmented-control';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Basic} from './examples/Basic';
+import {Disabled} from './examples/Disabled';
+import {TextAndIcon} from './examples/TextAndIcon';
+import {TextOnly} from './examples/TextOnly';
+import {Sizes} from './examples/Sizes';
+import {Vertical} from './examples/Vertical';
+import {RTL} from './examples/RTL';
+import {Dynamic} from './examples/Dynamic';
+import {
+  SegmentedControlModelConfigComponent,
+  SegmentedControlStateComponent,
+  SegmentedControlEventsComponent,
+} from './SegmentedControlModel.splitprops.tsx';
+
+<Meta title="Preview/Segmented Control/React" component={SegmentedControl} />
+
+# Canvas Kit Segmented Control
+
+Segmented Control is a
+[compound component](/getting-started/for-developers/resources/compound-components/) represents a
+linear group of two, three or four buttons allowing select speciic value. This is a controlled
+component.
+
+[> Workday Design Reference](https://design.workday.com/components/buttons/action-bar)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+### Basic Example
+
+In a basic example of an `SegmentedControl` there are four icon buttons in group. Each button paired
+with tooltip to explain the button function in text. Default values of `SegmentedControl` are
+specified for size (as `medium`) and orientation (as `horizontal`). All buttons should have the same
+type amd only one of them can be active at a time.
+
+`SegmentedControl` includes a container `SegmentedControl` component and the following subcomponent:
+`SegmentedControl.List` which should contains `SegmentedControl.Item`.
+
+<ExampleCodeBlock code={Basic} />
+
+### Variations Example
+
+`SegmentedControl` can be used with diferent buttons: text only, icon left or icon only.
+
+#### Text And Icon
+
+<ExampleCodeBlock code={TextAndIcon} />
+
+#### Text And Icon
+
+<ExampleCodeBlock code={TextOnly} />
+
+#### Icon Only (vertical)
+
+`SegmentedControl` icon only buttons can be placed horizontally or vertically.
+
+<ExampleCodeBlock code={Vertical} />
+
+### Sizes Example
+
+`SegmentedControl` has three different size: `small`, `medium` (default size) and `large`.
+
+<ExampleCodeBlock code={Sizes} />
+
+### Disabled Example
+
+``SegmentedControl`can be disabled by adding`disabled` prop to the whole container. It will also
+disable all buttons.
```

Comment:
Instead of "It will also disable all buttons." should it say "This will disable all buttons"